src/server.ts

```diff
@@ -28,8 +28,8 @@ export const app = express()
 // //////////////////////////////
 //
 //
-
-const AllowedOrigins = ['http://localhost:3000', 'https://boldo.penguin.software']
+const corsList = process.env.BOLDO_CORS || "http://localhost:3000"
```

Comment:
don't give fallback values for env variables. add `!` to assume them.
Also: you should use env variables directly if only referenced once instead of assigning an extra variable.

This line should look like this then:
`const AllowedOrigins = process.env.BOLDO_CORS!.split(",")`